src/components/Pagination.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import Button from './Button';
+
+const PaginationItem = ({ pageNumber, isActive, onChange }) => {
```

Comment:
Since this component is only gonna be used here, so I did not bother to create another file for it

Reply:
Also, the pagination item does not need to know how to compute if it is active or not, it should come from the the pagination parent as a flag